content/tools/cli/Cli-tool.md

```diff
@@ -0,0 +1,107 @@
+<!--
+title: "CLI Tool"
+description: "Instructions for installing the CLI tool for library analysis"
+tags: "tools cli library install"
+-->
+  
+Use Contrast's command line interface (CLI) tool for software composition analysis (SCA) to gather and send library dependency information for your applications to the Contrast UI. 
```

Comment:
I think we ought to expand on this and improve the messaging around why someone would want to use the SCA tool. Our agents collect library dependency information for applications as well so we ought to distinguish the two.

What about something like the following...

Contrast's command line interface (CLI) tool can perform software composition analysis (SCA) on your application. The SCA CLI tool is able to gather data on your application while it is in its `development` phase that is typically not available when your application is in the `deployment` phase. Using the data captured by the SCA CLI tool we are able to provide you a more detailed view of your applications in the Contrast UI.



Thoughts?

Reply:
@kaitlynmortimer In regards to the following comment you left on line 14 of `content/tools/cli/Cli-tool.md`:

> Can we add a couple more sentences here about how the tool works?

Will my above comment/additional content work? Or are you thinking of something else?

Reply:
Yep, that looks good!

Reply:
Agree and good to point out its in the development stage